include/WCSimTrajectory.hh

```diff
@@ -156,7 +155,7 @@ public: // with description
   // M Fechner : new saving mechanism
   G4bool SaveIt;
   G4bool producesHit;
-  G4String creatorProcess;
+  ProcessType_t creatorProcess;
```

Comment:
I am afraid this change of data type here will break the code somewhere, e.g. https://github.com/WCSim/WCSim/blob/575d4902df3253cf2725ed10d6e06c3a33f75981/src/WCSimEventAction.cc#L1396

I prefer to keep the original code in this class, and only do the conversion in `WCSimEventAction`

Reply:
👍